packages/theme_tailor/example/lib/nullable_params.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/material.dart';
+import 'package:theme_tailor_annotation/theme_tailor_annotation.dart';
+
+part 'nullable_params.tailor.dart';
+
+@tailor
+class _$NullableParams {
```

Comment:
I think it would be good to test it somehow, probably simplest test would be to create this class only specyfying `anotherProperty` and left `nullableProperty` unassigned (this would faild if it was generated as all-required)